bot.go

```diff
@@ -227,7 +227,12 @@ func (b *Bot) Run() error {
 //
 // If you need complete control over the regular expression, e.g. because you
 // want the patter to match only a substring of the message but not all of it,
-// you can use Bot.RespondRegex(…).
+// you can use Bot.RespondRegex(…). For even more control you can also directly
+// use Brain.RegisterHandler(…) with a function that accepts ReceiveMessageEvent
+// instances.
+//
+// If multiple matching patterns are registered, only the first registered
```

Comment:
I think this should say "... only the first matched handler is executed" (not the first registered)

Reply:
Thanks for the feedback, gonna update this bit of the documentation to clarify the intended behavior :+1:

Reply:
How about:

```
// If multiple matching patterns are registered, only the first matching handler
// is executed (ordered by sequence of handler registration).
```

I still want to make clear that it is important in what order you register your handlers but I'n not sure the changed comment is actually simpler to read while it still contains the same info. WDYT?